runelite-client/src/main/java/net/runelite/client/plugins/worldmap/TeleportLocationData.java

```diff
@@ -115,6 +115,10 @@ enum TeleportLocationData
 	CHAOS_TEMPLE(TeleportType.JEWELLERY, "Burning Amulet" , "Chaos Temple (lvl 15)", new WorldPoint(3234, 3637, 0), "burning_amulet_teleport_icon.png"),
 	BANDIT_CAMP(TeleportType.JEWELLERY, "Burning Amulet" , "Bandit Camp (lvl 17)", new WorldPoint(3038, 3651, 0), "burning_amulet_teleport_icon.png"),
 	LAVA_MAZE(TeleportType.JEWELLERY, "Burning Amulet" , "Lava Maze (lvl 41)", new WorldPoint(3028, 3840, 0), "burning_amulet_teleport_icon.png"),
+	ALDARIN(TeleportType.JEWELLERY, "Pendant of Ates", "Aldarin", new WorldPoint(1425, 2995, 0), "pendant_of_ates_icon.png"),
+	DARKFROST(TeleportType.JEWELLERY, "Pendant of Ates", "The Darkfrost", new WorldPoint(1490, 3284, 0), "pendant_of_ates_icon.png"),
+	RALOS_RISE(TeleportType.JEWELLERY, "Pendant of Ates", "Ralos' Rise", new WorldPoint(1459, 3137, 0), "pendant_of_ates_icon.png"),
+	TWILIGHT_TEMPLE(TeleportType.JEWELLERY, "Pendant of Ates", "Twilight Temple", new WorldPoint(1667, 3222, 0), "pendant_of_ates_icon.png"),
```

Comment:
I don't have a pendant to test this with, can someone who does please confirm these spots?